crates/neon/src/types_impl/mod.rs

```diff
@@ -1154,6 +1155,47 @@ impl JsFunction {
     }
 }
 
+pub(crate) unsafe fn call_local<'a, 'b, C: Context<'a>, T, AS>(
+    cx: &mut C,
+    callee: raw::Local,
+    this: Handle<'b, T>,
```

Comment:
Nit, it's a little odd for this to take a `Handle` when `callee` is a `raw::Local`. Can these both be handles or both `Local`?

Reply:
I moved it into a method of `ValueInternal`, so it doesn't feel as odd:

```rust
    unsafe fn try_call<'a, 'b, C: Context<'a>, T, AS>(
        &self,
        cx: &mut C,
        this: Handle<'b, T>,
        args: AS,
    ) -> JsResult<'a, JsValue>
    where
        T: Value,
        AS: AsRef<[Handle<'b, JsValue>]>,
    { /* ... */ }

// ...

unsafe { self.try_call(cx, this, args) }
```